src/test/java/org/cactoos/iterable/RangeOfTest.java

```diff
@@ -72,35 +72,35 @@ public Integer apply(
                 )
             ),
             Matchers.contains(1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89)
-        );
+        ).affirm();
     }
 
     @Test
     public final void testLongRange() {
-        MatcherAssert.assertThat(
-            "Can't generate a range of long",
+        new Assertion<>(
+            "Must generate a range of long",
             new ListOf<>(
                 new RangeOf<>(1L, 5L, value -> ++value)
             ),
             Matchers.contains(1L, 2L, 3L, 4L, 5L)
-        );
+        ).affirm();
     }
 
     @Test
     public final void testCharacterRange() {
-        MatcherAssert.assertThat(
-            "Can't generate a range of characters",
+        new Assertion<>(
+            "Must generate a range of characters",
             new ListOf<>(
                 new RangeOf<>('a', 'c', value -> ++value)
             ),
             Matchers.contains('a', 'b', 'c')
-        );
+        ).affirm();
     }
 
     @Test
     public final void testLocalDateRange() {
-        MatcherAssert.assertThat(
-            "Can't generate a range of local dates.",
+        new Assertion<>(
+            "Must generate a range of local dates.",
```

Comment:
@baudoliver7 Probably we can move `final` modifier to class definition

Reply:
@andreoss Yes. I removed also `public`.

Reply:
@andreoss Yes. I removed also `public`.